### DISCUSSION TITLE
CHROOT Installation

### DISCUSSION THREAD

**Discussion by tichzvidzayi** [ltsp/ltsp#240] [Q&A]
Hi All,

I have been trying to install ltsp in chroot mode but with no luck is there any step by step guideline on how to do it?   

.........................................................................................................................................................................
Below I am having a could not load kernel error 

kvm -m 512 -kernel img_name/vmlinuz -initrd img_name/initrd.img \
        -append "rw root=/dev/nfs nfsroot=192.168.0.10:/srv/ltsp/img_name"_

.......................................................................................................................................................................
Using the other methods on https://github.com/ltsp/community/wiki/chroots
with debootstrap even after replacing the focal the folders I am supposed to navigate to don't exist.

Apologies for the bulky words and questions. I am still new

---------------------------------------------------------------------------------------------
If the issue you're about to report is likely to result in LTSP code or documentation changes, file an LTSP issue instead:
https://github.com/ltsp/ltsp/issues

**Comment by alkisg** [comment]
Chroots are best for advanced terminal users... why not use a VirtualBox or KVM virtual machine instead?
I've written a quick how-to in https://github.com/ltsp/community/wiki/chroots, but noone has provided any step-by-step tutorials. I'm sure you can find hundreds of tutorials on the Internet though, as maintaining chroots isn't specific to LTSP.

Just use any tutorial from the Internet, but put them in /srv/ltsp/image, and LTSP will find them.
